src/styles/main.scss

```diff
@@ -1,3 +1,17 @@
+* {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+}
+
+title {
+  display: none;
+}
```

Comment:
why do you need it?

Reply:
I had an issue with the title displayed at the top of the page, and tests also were counting it and failing, and as it is necessary I just use display: none; now an issue seems to be solved, new commit without this code